cmd/admin-handlers-config-kv.go

```diff
@@ -201,6 +207,18 @@ func setConfigKV(ctx context.Context, objectAPI ObjectLayer, kvBytes []byte) (re
 		return
 	}
 
+	// Check if callhome getting enabled, if so set the subnet proxy value to logger_webhook's subnet proxy
+	// Do this only if logger_wenhook's subnet proxy is not yet set
+	if result.SubSys == "callhome" && result.Cfg["callhome"]["_"].Get("enable") == "on" {
```

Comment:
The `callhome` subsystem is related to configuring the daily diagnostics callhome, and not related to logs. So this condition is not required. Instead, we need to check if the config being modified is for the `subnet` subsystem.